app/js/components/FiatSelectorForm.jsx

```diff
@@ -0,0 +1,103 @@
+import React, {Component, Fragment} from 'react';
+import PropTypes from 'prop-types';
+import {FormGroup, FormFeedback} from 'reactstrap';
+import {Typeahead} from 'react-bootstrap-typeahead';
+import '../../../node_modules/react-datetime/css/react-datetime.css';
+import '../../css/Form.scss';
```

Comment:
may be we should import those 2 files globally? in `index.js`?

Reply:
Good idea!